scripts/lint/logger.ts

```diff
@@ -56,7 +56,7 @@ const prettyPrint = (
         color.dim(String(startLine + 1)),
         color.dim(lines[startLine]),
       ),
-      sprintf("%*s╰─► %s", pad, "", error),
+      sprintf("%*s╰─► %s", pad, "", error.split('\n').map((str, i) => i === 0 ? str : " ".repeat(pad + 4) + str).join('\n')),
```

Comment:
This change pads multiline error messages to have the same indentation as the first line.